jenkins/test_drop_in_envs.groovy

```diff
@@ -22,6 +22,8 @@ node('release-dev && memory-intense'){
         popd
     '''.stripIndent()
 
+    createInitialAdminUser()
```

Comment:
Maybe leave a comment that explains which env var gets exported (`APP_ADMIN_API_KEY`) and a hint about where this groovy function comes from.  That way there are no mysteries for our future selves.

Reply:
Yeah, good point. 
Where does it come from?